sql/updates/characters/6.x/2016_MM_DD_00_characters.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `character_pet` ADD COLUMN `specialization` SMALLINT UNSIGNED NOT NULL DEFAULT '0' AFTER `abdata`;
```

Comment:
Added in https://github.com/TrinityCore/TrinityCore/pull/17058/commits/d43d54e99750b9fb1af27e8bca26957f49e94441 .